admin/workspace-management/process-logging.md

```diff
@@ -65,10 +65,25 @@ To view the process logs from a specific user or workspace, you can use your
 cloud provider's log viewer, or you can use `kubectl` to print the logs:
 
 ```bash
-kubectl logs \
-  -l "com.coder.username=zac" \        # Filter by the user "zac"
-  -l "com.coder.workspace.name=code" \ # Filter by the workspace "code"
-  -c exectrace                         # Only show logs from the sidecar
+kubectl logs /
```

Comment:
these should be `\` - the backslash is needed to escape the newline, which is how these can be multi-line commands

```suggestion
kubectl logs \
```